src/Microsoft.Fast.Components.FluentUI/Components/FluentAnchoredRegion.razor

```diff
@@ -0,0 +1,103 @@
+<fluent-anchored-region anchor="@Anchor"
+                        viewport="@Viewport"
+                        horizontal-positioning-mode="@HorizontalPositioningMode.ToAttributeValue()"
+                        horizontal-default-position="@HorizontalDefaultPosition.ToAttributeValue()"
+                        horizontal-inset="@HorizontalInset"
+                        horizontal-threshold="@HorizontalThreshold"
+                        horizontal-scaling="@HorizontalScaling.ToAttributeValue()"
+                        vertical-positioning-mode="@VerticalPositioningMode.ToAttributeValue()"
+                        vertical-default-position="@VerticalDefaultPosition.ToAttributeValue()"
+                        vertical-inset="@VerticalInset"
+                        vertical-threshold="@VerticalThreshold"
+                        vertical-scaling="@VerticalScaling.ToAttributeValue()"
+                        auto-update-mode="@AutoUpdateMode.ToAttributeValue()"
+                        @attributes="AdditionalAttributes">
+    @ChildContent
+</fluent-anchored-region>
+
+@code
+{
+    /// <summary>
+    /// The html id of the HTMLElement used as the anchor around which the positioning region is placed. This must be set for the component's positioning logic to be active.
+    /// </summary>
+    [Parameter]
+    public string Anchor { get; set; }
+
+    /// <summary>
+    /// The ID of the HTMLElement to be used as the viewport used to determine available layout space around the anchor element. If unset the parent element of the anchored region is used.
+    /// </summary>
+    [Parameter]
+    public string Viewport { get; set; }
+
+    /// <summary>
+    /// Default is 'uncontrolled'.
+    /// </summary>
+    [Parameter]
+    public Positioning? HorizontalPositioningMode { get; set; } = Positioning.Uncontrolled;
+
+    /// <summary>
```

Comment:
Does this syntax surface the summary to editors like VSCode? I tried this syntax in the DesignSystemProvider PR but could not get this documentation to surface to the editor.

Reply:
It does in VS and it should also work in VS Code. Are you using the latest C# for VS Code extension (https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp)?  

![image](https://user-images.githubusercontent.com/1761079/136090671-039b1fed-d034-4f7e-b096-0e8e6fa3ac41.png)

Reply:
Good question, I'm not sure. I'll play around with it. Good to know though, thanks!